weave

```diff
@@ -1231,10 +1211,10 @@ launch() {
     fi
 
     # Create a data-only container for persistence data
-    if ! docker inspect -f ' ' $DB_CONTAINER_NAME > /dev/null 2>&1 ; then
-       protect_against_docker_hang
-       docker create -v /weavedb --name=$DB_CONTAINER_NAME \
-           --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
+    if ! util_op container-state $DB_CONTAINER_NAME > /dev/null 2>&1 ; then
+        protect_against_docker_hang
```

Comment:
spurious extra space

Reply:
An extra space because it wasn't aligned.  I can revert if you want that in a different commit.  I noticed it when adding the line below it...

Reply:
doesn't matter